### PR TITLE
update Create react app scripts to v2

### LINKED ISSUE #1753: Use CRA@next
We need to move to CRA@next which has webpack4 and babel7. Fixes a few issues and helps us moving forward development wise.

### PR DESCRIPTION
**Pull request by farmasek**
Hi, so CRA is at its latest beta, ready to testing. 

fixes #1753

### CODE REVIEW

File: src/components/Match/Vision/index.jsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const Styled = styled.div`
   }
 }
 `;
-
+/* eslint-disable jsx-a11y/anchor-is-valid */
```
Comment (howardchung) [inline src/components/Match/Vision/index.jsx]: we should just make this a warning in eslint config

**Comment by howardchung** [comment]
2.0.4 is the latest bugfix

File: src/components/Match/Vision/index.jsx (hunk 1 above)
Comment (farmasek) [reply src/components/Match/Vision/index.jsx]: normally it is a warning, but in CI build it fails beacause warnings are treated as errors. Interesting is, that it only fails at this warning, there are several more in the project which build ignores.

**Comment by dartharaknus** [comment]
What about the polyfills @gu3st ?

**Comment by gu3st** [comment]
I'll test it later and see if those are redundant.

**Pull request merged**